api/envoy/extensions/filters/http/simple_http_cache/v3alpha/BUILD

```diff
@@ -0,0 +1,14 @@
+# DO NOT EDIT. This file is generated by tools/proto_format/proto_sync.py.
+
+load("@envoy_api//bazel:api_build_system.bzl", "api_proto_package")
+
+licenses(["notice"])  # Apache 2
+
+api_proto_package(
```

Comment:
i just copied the file from `api/envoy/extensions/filters/http/cache/v3alpha/BUILD` to here

given the confusion between these two - ie `cache`  <> `simple_http_cache` im not at all sure where the proto should be moved to and/or what/where the BUILD should be